Stratis.BreezeD/Program.cs

```diff
@@ -26,6 +26,7 @@ public static void Main(string[] args)
             var apiUri = args.GetValueOf("apiuri");
             var isTestNet = args.Contains("-testnet");
             var isStratis = args.Contains("stratis");
+            var agent = "Breeze(spv)";
```

Comment:
I think the agent should be "Breeze", no need to specify spv.

Reply:
I added spv so observers in the network that dont know what breeze is will get an idea of the nodes purpose. (It applies mostly for the bitcoin netwrok).

Reply:
Ok I will change it for now and remove the `spv`